modules/passkey/contracts/SafeWebAuthnSignerProxyFactory.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Can we please not rename this file? I think it is still a `SignerFactory`. The fact it uses proxy contracts is an internal detail that does not need to be reflected in a change of contract name IMO.

Reply:
I did it because we have `SafeProxyFactory` for safe creating safe accounts. Will revert name change though.

Reply:
https://github.com/safe-global/safe-modules/pull/370/commits/fa594dc1fd83e353ae7c362e4f221c963c717dbb